src/components/blocks/object-image-overlay/_object-image-overlay.js

```diff
@@ -304,6 +300,37 @@ const oicInit = () => {
             oic.advance(true);
           }
         };
+
+        oic.ontouchstart = (e4) => {
+          const startXY = [e4.touches[0].pageX, e4.touches[0].pageY];
+          oic.ontouchmove = (e5) => {
+            const deltaXY = [e5.touches[0].pageX - startXY[0], e5.touches[0].pageY - startXY[1]];
+            if (Math.abs(deltaXY[0]) > Math.abs(deltaXY[1])
+              && (
+                (deltaXY[0] < 0 && oic._index < oicSeeds.length - 1)
+                || (deltaXY[0] > 0 && oic._index > 0)
+              )) {
+              if (Math.abs(deltaXY[0]) < 74) {
```

Comment:
Magic number here with no indication of how it is derived.

Reply:
yes, could do with commenting generally. that's just 74px, a minimum touchmove distance to activate a full swipe.